.jenkins/infrastructure/docker/build_linux_docker_images.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+OECI_LIB_VERSION = env.OECI_LIB_VERSION ?: "master"
+oe = library("OpenEnclaveCommon@${OECI_LIB_VERSION}").jenkins.common.Openenclave.new()
+
+AGENTS_LABEL = params.AGENTS_LABEL
```

Comment:
You do not want a default here, is that correct?

Reply:
Yeah, I wanted the agent to be explicitly defined - we have defaults in Jenkins, so this shouldn't be an issue.